e2e/smoke/big/dev_e2big_test.sh

```
@@ -0,0 +1,64 @@
+WORK="${TEST_TMPDIR}/devrun"
+mkdir -p "${WORK}"
+
+# The dev launcher resolves its runfiles as "<script path>.runfiles". Running
+# as a data dep of this test, it has no runfiles tree of its own — but this
+# test's TEST_SRCDIR is a superset (data runfiles are merged). Copy the script
+# and fake the .runfiles link next to the copy.
+cp "${LAUNCHER}" "${WORK}/dev.sh"
+ln -sfn "${TEST_SRCDIR}" "${WORK}/dev.sh.runfiles"
+
+# cd so the launcher's relative mktemp dirs land in the test tmpdir.
+cd "${WORK}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)